integration-test-standard/src/test/java/com/ericsson/bss/cassandra/ecaudit/integration/standard/ITDataAudit.java

```diff
@@ -89,8 +89,8 @@ public static void afterClass()
         return new Object[]{
             new Object[]{ "CREATE KEYSPACE IF NOT EXISTS dataks WITH REPLICATION = {'class' : 'SimpleStrategy', 'replication_factor' : 1} AND DURABLE_WRITES = false", "create", "data/dataks" },
             new Object[]{ "CREATE TABLE IF NOT EXISTS dataks.tbl (key int PRIMARY KEY, value text)", "create", "data/dataks" },
-            new Object[]{ "CREATE INDEX IF NOT EXISTS idx ON dataks.tbl (value)", "alter", "data/dataks/tbl" },
-            new Object[]{ "CREATE MATERIALIZED VIEW IF NOT EXISTS dataks.viw AS SELECT value FROM dataks.tbl WHERE value IS NOT NULL AND key IS NOT NULL PRIMARY KEY (value, key)", "alter", "data/dataks/tbl" },
+            new Object[]{ "CREATE INDEX IF NOT EXISTS idx ON dataks.tbl (value)", "alter", "data/dataks/idx" },
+            new Object[]{ "CREATE MATERIALIZED VIEW IF NOT EXISTS dataks.viw AS SELECT key, value FROM dataks.tbl WHERE value IS NOT NULL AND key IS NOT NULL PRIMARY KEY (value, key)", "alter", "data/dataks/tbl" },
```

Comment:
Should this detail be fixed in older branches as well?

Reply:
No, I don't think so. This is a change in how materialized views work in 4.0 and its documented in the news file. Before 4.0 the partion key was added to the select statement used by the view but in 4.0 you must add partion key to the statement explicitly.

Reply:
Check!